### DISCUSSION TITLE
inav osd menu

### DISCUSSION THREAD

**Discussion by ozkancikikci** [iNavFlight/inav#8117] [General]
hi
I'm using f411wte,tx16. I can't enter the inav osd menu. mode 2 
 what is the procedure.

![Ekran Alıntısı](https://user-images.githubusercontent.com/107365651/173247780-3f7125aa-c8b2-4c7a-b086-368fccdd2e36.JPG)

**Comment by MrD-RC** [comment]
Are you sending the yaw channel from the transmitter?
